docs/wiki/feature/advanced-fatigue.md

```diff
@@ -0,0 +1,99 @@
+- **Injuries:** Pain and blood volume
+- **Carrying:** Both units and objects
+- **Environment temperature**
+
+
+### 1.3 Effects of High Fatigue
+
+Depending on how much your character is fatigued you will experience various visual, audible and physical symptoms:
+
+- Breathing sounds (with different intensity and frequency)
+- Black flashing border when hitting the bodies limits
+- Not being able to sprint
+- Not being able to jog
+- Slowed down movement whilst swimming or diving
+- Weapon sway (majorly reduced when crouched or prone)
+
+
+### 1.4 Stamina Bar
+
+ACE provides a stamina bar similar to the one in vanilla Arma 3. You can enable and disable it through your ACE options menu. If you want to move the stamina bar, simply move the vanilla one in the GUI game options, the Advanced Fatigue one will move to the same location (this requires a restart though).
+
+The Advanced Fatigue stamina bar has a few changes compared to the vanilla one to improve both immersion and user feedback, inspired by Dslyecxi's custom stamina bar. It will initially have a very low opacity which rises with the loss of stamina. Additionally it will first turn from white to orange and bright red afterwards.
+
+
+
+## 2. Simulation Details
+
+Since Advanced Fatigue is based on real life biological systems we will explain some of them in more detail here.
+
+
+### 2.1 How Human Stamina Works
+
+Human stamina is based on the availability of adenosine triphosphate (ATP), which is a molecule used by the body to transfer power. Small amounts of ATP are already stored readily available within muscle cells and larger amounts are generated by several pathways, which then transfer the ATP through blood to the muscles. There are quite a few pathways which generate and store ATP, so instead of simulating all of them, we merge them into 3 generic pathways.
+
+
+### 2.2 ATP Pathways
+
+Each pathway has different properties: They store different amounts of ATP and can provide and regenerate different amounts of ATP per second. You can compare this to having three differently sized barrels filled with water, each having a differently sized faucet and opening at the top to refill it with water.
+
+
+#### 2.2.1 Anaerobic Pathway
+
+This pathway is essentially responsible for your short term stamina. It has a very small capacity of ATP available (2300 mmol), which is can provide a lot of ATP very fast (13.3 mmol/s), but also replenished very quickly (56.7 mmol/s). This pathway is pretty much what limits how long you can jog or sprint in one go, which is why we use it as an indicator for the stamina bar.
+
+
+#### 2.2.2 Aerobic Pathways
+
+The two aerobic pathways store way bigger amounts of ATP than the anaerobic one, the first one stores over 1700 times as much ATP (4000 mol) and the second one still over 36 times as much ATP (84 mol) as the aerobic pathway. However, they can also provide (13.3 and 16.7 mmol) and replenish (6.6 and 5.83 mmol) a lot less ATP then the aerobic pathway.
```

Comment:
The two aerobic pathways store much larger amounts of ATP than the anaerobic one.
The first pathway stores over 1700 times as much ATP (4000 mol), while the second one still has over 36 times as much ATP (84 mol) than the anaerobic pathway. However, they also provide (13.3 and 16.7 mmol) and replenish (6.6 and 5.83 mmol) considerably less ATP than the aerobic pathway.